certified-connectors/Eigen Events Connector/apiDefinition.swagger.json

```diff
@@ -0,0 +1,298 @@
+        "description": "Details of the incoming event.",
+        "schema": {
+          "type": "object",
+          "x-ms-dynamic-schema": {
+            "operationId": "GetDynamicEventSchema",
+            "parameters": {
+              "eventIds": {
+                "parameter": "eventIds"
+              }
+            },
+            "value-path": "schema"
+          }
+        }
+      },
+      "post": {
+        "description": "This operation triggers when any of the events subscribed to occurs",
+        "summary": "When an event occurs",
+        "operationId": "CreateWebhook",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "eventIds",
+            "in": "path",
+            "required": true,
+            "x-ms-summary": "EventID",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "GetSupportedEvents",
+              "value-path": "value",
+              "value-collection": "events",
+              "value-title": "display"
+            },
+            "description": "Select Event Id",
+            "type": "number"
+          },
+          {
+            "name": "Body of Trigger Request",
+            "in": "body",
+            "description": "The body of Trigger-Creation request",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "required": [
+                "callbackUrl"
+              ],
+              "properties": {
+                "callbackUrl": {
+                  "title": "The callback URL",
+                  "x-ms-summary": "Callback URL",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Response to successfully created webhook.",
+            "schema": {
+              "$ref": "#/definitions/WebhookCreationResponse"
+            }
+          },
+          "default": {
+            "description": "Response to failed request.",
+            "x-ms-summary": "Failure response"
+          }
+        }
+      }
+    },
+    "/events/api/webhook/delete/{webhookId}": {
+      "delete": {
+        "description": "This action deletes a webhook you had created",
+        "summary": "Delete a webhook you created",
+        "operationId": "DeleteWebhook",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "webhookId",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "description": "The ID of the webhook to delete",
+            "x-ms-visibility": "important",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Response to successfully deleting a webhook."
+          },
+          "default": {
```

Comment:
@tonyscofieldkim Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Refer - [Documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-responses)